.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
adding a gitignore for the root dir

Reply:
Good question. This is definitely the faster/copy-paste way of doing it.  I like it because the one in the mrtt-ui folder can just worry about itself where as editing the one at root could implicate many projects. 

Alternatively, if we did go for only one .gitignore, we would likely be explicitly duplicating a lot and putting the project folder in front (eg `mrtt-ui/coverage`) dangerously(?) using wildcards.

Im leaning on the multiple files to scope stuff more reliably (and make it easier to read - at least for my brain), but I have little experience configuring a monorepo. @JBurkinshaw @ghelobytes